setup.cfg

```diff
@@ -1,68 +1,81 @@
 [metadata]
 name = halotools
-version = 0.8
-description = Package designed to analyze N-body simulations and constrain models of cosmology and galaxy evolution.
 author = Andrew Hearin
 author_email = ahearin@anl.gov
-license = BSD
+license = BSD 3-Clause
+license_files = licenses/LICENSE.rst
 url = http://astropy.org/
+description = Package designed to analyze N-body simulations and constrain models of cosmology and galaxy evolution.
+long_description = file: README.rst
+long_description_content_type = text/x-rst
 edit_on_github = True
 github_project = astropy/halotools
 
+[options]
+zip_safe = False
+packages = find:
+python_requires = >=3.9
+setup_requires = setuptools_scm
+install_requires =
+    astropy numpy scipy requests beautifulsoup4 cython
```

Comment:
These need to be one line a package.

Reply:
Thanks, yes, good catch. My cython is now (starting to) build so this got me farther down the road.